docs/distributions/ubuntu/upgrade.md

```diff
@@ -0,0 +1,55 @@
+# Upgrade
+
+Recommendations:
+
+* Perform upgrade from LTS to LTS.
+
+Warnings:
+
+* If you have important data and workspace, make sure that you have backup.
+* Make sure you have enough disk space.
+* **Ensure that you met all the below requirements so that you can recover in case of failed upgrade.**
+
+# Hardware Requirements
+
+* USB-C to Ethernet adapter **OR** Smartphone capable of USB tethering to provide internet in case of emergency
+* A cable that can connect one of the above objects to your Mac
+
+# Kernel Requirements
+
+Check the latest official Ubuntu release and find its kernel version. You need to install the same or higher custom kernel version for your current Ubuntu.
```

Comment:
I don't think we need to install the custom kernel again. It shouldn't be removed imo.

Reply:
I think we need to have custom_kernel_version >= new_lts_ubuntu_kernel_version to ensure that after the upgrade, the new ubuntu will work without problem. I assume the new ubuntu release doesn't work well with older kernels versions.
Previously installed kernels are preserved.

Reply:
I added a few words to the wiki to make this more clear.